### PR TITLE
Fixed #31029 -- Used more specific links to RFCs.

### PR DESCRIPTION
**Pull request by bmispelon**
ticket-31029

### CODE REVIEW

File: docs/ref/settings.txt [hunk 1]
```diff
@@ -3108,8 +3108,8 @@ Whether to use ``HttpOnly`` flag on the session cookie. If this is set to
 cookie.
 
 HttpOnly_ is a flag included in a Set-Cookie HTTP response header. It's part of
-the :rfc:`6265` standard for cookies and can be a useful way to mitigate the
-risk of a client-side script accessing the protected cookie data.
+the :rfc:`6265#section-5.3` standard for cookies and can be a useful way to
```
Comment (felixxm) [inline docs/ref/settings.txt]: I think this should point to the `section-4.1.2.6` (like in the ` docs/ref/request-response.txt`) or in both cases we should leave `` :rfc:`6265` ``.

File: docs/ref/validators.txt [hunk 2]
```diff
@@ -154,7 +154,8 @@ to, or in lieu of custom ``field.clean()`` methods.
     an error code of ``'invalid'`` if it doesn't.
 
     Loopback addresses and reserved IP spaces are considered valid. Literal
-    IPv6 addresses (:rfc:`2732`) and unicode domains are both supported.
+    IPv6 addresses (:rfc:`2732#section-2`) and unicode domains are both
+    supported.
```
Comment (felixxm) [inline docs/ref/validators.txt]: This RFC is obsolete I think we should use `rfc3986#section-3.2.2`.

**Comment by felixxm** [comment]
@bmispelon Thanks :+1: I reorganized commits and left two comments. Can you take a look?

File: docs/ref/settings.txt (hunk 1 above)
Comment (bmispelon) [reply docs/ref/settings.txt]: You're right. `section-5.3` was a bad copy/paste I think.

I've updated the PR to point to section 4.1.2.6 as suggested.

File: docs/ref/validators.txt (hunk 2 above)
Comment (bmispelon) [reply docs/ref/validators.txt]: I've updated the RFC number (and paragraph) as suggested.

I've also checked to see if RFC 2732 was referenced anywhere else but that doesn't appear to be the case.

**Comment by bmispelon** [comment]
Oops, I was a bit too blunt with my `git push -f` and managed to lose the changes you'd made to the branch.

I think I've managed to fix it now. I've restored your commits and added the two changes you'd requested on top of it.

Sorry for the mess, I'll be more careful next time :man_facepalming:

**Comment by felixxm** [comment]
@bmispelon np, Thanks for updates :+1:

**Pull request merged**